### ISSUE TITLE
Community

### ISSUE THREAD

**Issue by beefsack** [google/xi-editor#35]
I'm incredibly excited to see this editor grow, do you have any thoughts about a place where people can discuss?  Maybe an IRC channel or mailing list?

I've thrown together an unofficial subreddit: https://www.reddit.com/r/xi_editor/

I also idle a bit on #xi on freenode (irc://irc.freenode.net/xi)

**Comment by stoivo** [comment]
[gitter.im](https://gitter.im/)?

**Comment by raphlinus** [comment]
It's an excellent question. I've thought about it a little and don't have a clear preference. I'd kinda lean toward a [Discourse](https://www.discourse.org/) instance because it seems explicitly designed for this kind of thing, and the ones I know ([CommonMark](http://talk.commonmark.org/) and the Rust ones) seem to function pretty well. Mailing lists seem pretty clunky and need care and feeding to keep them from rotting. IRC I think would not be ideal because my attention is bursty, I'd like to easily be able to catch up on activity (I know there are solutions such as logs, but I never read the #rust ones).

So, basically, suggestions welcome. What has worked well for similar needs?

**Comment by raphlinus** [comment]
Another consideration: I've got a bunch of long-form content queued up (I call it my "rope science" series) so ideally the community platform would either be a good place to host that, or I post those as static github pages and link to a conversation. IRC and IRC-like approaches don't seem like they would work well for that.

**Comment by Dushistov** [comment]
Why not create google group, it works as web forum and also as mailing list, plus I suppose google filter spam for them.

**Comment by davelab6** [comment]
Mailman3/Hyperkitty is fine, Discourse is good, but some projects now use Github Issues as a forum ;) eg https://github.com/ipfs/faq

**Comment by beefsack** [comment]
GitHub pages for long form (and could potentially serve as a web page one day) and Discourse for discussion seems like a nice combo.

IRC channels are trivial to spin up so I think they're usually worth it for technical subjects, I'm sure we've got more than a handful of Freenode idlers here.

**Comment by beefsack** [comment]
I've created an unofficial subreddit at https://www.reddit.com/r/xi_editor/ (/r/xi was too short unfortunately.)

**Comment by shymega** [comment]
I would also be interested in an IRC channel for Xi Editor - it'd be a good place for technical and support discussions.

@beefsack, is the IRC channel still active? I just popped in, and couldn't see anyone there, but it looks like the channel is registered though. Thanks.

**Comment by beefsack** [comment]
@shymega I idled in there for a while but I'm not always on IRC.  Was thinking about chucking a bot in there so it wasn't empty but haven't gotten around to anything yet.

**Comment by brendanzab** [comment]
The advantage of Gitter is the asynchrony (without the complexities of setting up bouncers). A bunch of Rust projects use it too - Diesel, Gfx, Gluon...

**Comment by shymega** [comment]
Hello,

Michael Alexander <notifications@github.com> writes:

> @shymega I idled in there for a while but I'm not always on IRC.  Was
> thinking about chucking a bot in there so it wasn't empty but haven't
> gotten around to anything yet.
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/google/xi-editor/issues/35#issuecomment-238112451

Yeah, you came in for a while yesterday, I think - I was asleep then, so
I missed you coming in at all.

Just a suggestion, the #xi channel on FN appears to have been registered
in 2009.

Perhaps to avoid conflicts with the owner (?), we should emulate our
reddit approach and register #xi-editor instead.

This also has the added advantage of keeping a uniform identifier for
our various social outlets.

Alternatively, we could host an IRC channel on the Mozilla network
instead.

-- 
Best regards,
Dom Rodriguez

**Comment by shymega** [comment]
Hello,

Brendan Zabarauskas <notifications@github.com> writes:

> The advantage of Gitter is the asynchrony (without the complexities of
> setting up bouncers). A bunch of Rust projects use it too - Diesel,
> Gfx, Gluon...
>
> ---
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/google/xi-editor/issues/35#issuecomment-238153433

It's worth mentioning that IRC has tools such as botbot.me,
iirc.

We could also setup our own bot to log as well as other channel
utilities.

Bouncers aren't always necessary for the logging aspect.

-- 
Best regards,
Dom Rodriguez

**Comment by cmyr** [comment]
I think we've mostly settled on irc.mozilla.org#xi for community stuff, although we may wish to revisit at some point.
